### PR TITLE
docs(zh-CN): enable social sign-in

### PR DESCRIPTION
**Pull request by gao-sun**
启用社交登录

close LOG-3123

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-social-sign-in.mdx [hunk 1]
```diff
@@ -2,4 +2,59 @@
...
+
+如果你之前没有添加过其他的登录方式，「启动二级登录」应该是关着的。打开它，并勾选「社交登录」。
+
+在显示的表格左列「社交连接器」中寻找你刚添加的连接器，并点按右侧的加号图标。
+
+现在你应该能看到，在预览中大大的「登录」按钮下面，显示着代表着该连接器身份提供商的 logo。
+
+如果你启用了多个社交连接器（显示在右列），你可以通过拖拽它们进行排序。
+
+点按「保存变更」使你的变更上线。
```
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-social-sign-in.mdx:47]: nit: better `点按「保存变更」使你的变更上线。` → `点按「保存变更」上线你的变更。`

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-social-sign-in.mdx [hunk 2]
```diff
@@ -2,4 +2,59 @@
...
+
+:::tip
+再次打开演示应用并尝试新的登录方式。
+:::
+
+:::note
+查看 [配置登录方式](../../recipes/customize-sie/configure-sign-in-methods.mdx) 来全面地了解如何将登录方式进行组合。
```
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-social-sign-in.mdx:54]: nit: better `将登录方式进行组合` → `组合各种登录方式`

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-social-sign-in.mdx [hunk 3]
```diff
@@ -2,4 +2,59 @@
...
+:::
+
+## 接下来
+
+- [启用短信或邮件验证码登录](./enable-passcode-sign-in.mdx) （如果你还没看过）
+- [探索更多](./further-readings.md)
```
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-social-sign-in.mdx:60]: nit: `探索更多` → `拓展阅读` / `进一步阅读` / `延展阅读` (seems more common)?

**Review by IceHe — APPROVED** [review]
Overall LGTM

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-social-sign-in.mdx (hunk 2 above)
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-social-sign-in.mdx:54]:
```suggestion
查看 [配置登录方式](../../recipes/customize-sie/configure-sign-in-methods.mdx) 来全面地了解如何组合各种登录方式。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-passcode-sign-in.mdx [hunk 4]
```diff
@@ -48,10 +48,19 @@ import ConnectorsWip from './fragments/_connectors-wip.md';
...
+现在你应该能看到，在预览中大大的「登录」按钮下面，显示着「通过手机号登录」或「通过邮箱登录」的字样。
 
 点按「保存变更」使你的变更上线。
 
 :::tip
 再次打开演示应用并尝试新的登录方式。
 :::
+
+:::note
+查看 [配置登录方式](../../recipes/customize-sie/configure-sign-in-methods.mdx) 来全面地了解如何将登录方式进行组合。
```
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-passcode-sign-in.mdx:60]:
```suggestion
查看 [配置登录方式](../../recipes/customize-sie/configure-sign-in-methods.mdx) 来全面地了解如何组合各种登录方式。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-social-sign-in.mdx (hunk 3 above)
Comment (IceHe) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-social-sign-in.mdx:60]: nit: `探索更多` → `拓展阅读` / `进一步阅读` / `延展阅读` more common?
```suggestion
- [进一步阅读](./further-readings.md)
```

Comment (gao-sun) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/tutorials/get-started/enable-social-sign-in.mdx:60]: prefer「延展阅读」, will update later

**Pull request merged**